ucsb-cs156-f24/proj-dining-f24-15
# MenuItemReview GET (by logged in user) endpoint

dokku: https://sophiaray2101-get-id.dokku-15.cs.ucsb.edu/

This PR is for the get by id endpoint. There are no parameters to input because the job is to get all menu item reviews that were completed by that specific user id. AKA, it automatically will take the ID of the logged in user and get any reviews from that user. Since the menuItem is not merged into main yet, and so no posts can be made, the response will be empty. However, there should be no errors and it should return an empty list (until other thing are merged).

What to test:
1. Make sure you cant get anything not logged in
2. make sure there are no errors 

<img width="1265" alt="Screenshot 2024-11-25 at 3 36 50 PM" src="https://github.com/user-attachments/assets/b64d57f9-291b-4922-80c0-764fe3de297e">

Closes #8
